src/test/java/com/skhu/luxuryshop/product/repository/ProductRepositoryTest.java

```diff
@@ -0,0 +1,40 @@
+package com.skhu.luxuryshop.product.repository;
+
+import com.skhu.luxuryshop.product.dto.ProductRequestDto;
+import com.skhu.luxuryshop.product.entity.ProductEntity;
+import org.junit.After;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class ProductRepositoryTest {
+    @Autowired
+    ProductRepository productRepository;
+    private ProductRequestDto product;
+
+    @BeforeEach
+    void setup() {
+        product = new ProductRequestDto("서비스Test", "서비스 컨텐트 테스트", 900, "아우터", "www.asd");
+    }
+
+    @After
+    public void cleanup() {
+        productRepository.deleteAll();
+        System.out.println("테스트 완료");
+    }
```

Comment:
```suggestion
    @BeforeEach
    void setUp() {
        product = new ProductRequestDto("서비스Test", "서비스 컨텐트 테스트", 900, "아우터", "www.asd");
    }

    @AfterEach
    public void tearDown() {
        productRepository.deleteAll();
    }
```
setup 카멜케이스 적용이랑 @After은 junit5부터 @AfterEach로 사용하구 메서드명 컨벤션은 tearDown입니다.
그리고 불필요한 디버깅로그는 찍지 않는게 좋습니다!